### PR TITLE
created a dynamic form for bank details and stored the data in bank details table

### PR DESCRIPTION
**Pull request by AJAYSINGHBISHT12**
Targets #{2972}
[2972]

### Description
created a dynamic form and saved its data in database in bank-details table

### Checklist:

- [x] I have performed a self-review of my own code.

### CODE REVIEW

**Comment by rahulpanwar16** [comment]
@AJAYSINGHBISHT12  There are a few suggestions that needs to be done when you are making the PR.

```In Assignees``` : Assign it to yourself before assigning to the reviewers.
```In Label``` : Put ready for review label before assigning it to the reviewers.
```In Reviewers``` : Assign it to team-portal-reviewers.

**Comment by AJAYSINGHBISHT12** [comment]
tableindatabasein
forminactual
dataisgettingsaved
this is how first dynamic form is created 
then the data is getting saved in database 
and then this saved icon is showing

**Comment by rahulpanwar16** [comment]
> tableindatabasein forminactual dataisgettingsaved this is how first dynamic form is created then the data is getting saved in database and then this saved icon is showing

@AJAYSINGHBISHT12 we want this UI of success toast message. Please change it 
Screenshot (99)

File: resources/views/settings/bank-details.blade.php [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+@section('content')
+    <div class="container">
+        <h4>Bank Details</h4>
+        <br>
+        <form action="{{ route('settings.bank-details.update') }}" method="POST">
+            @csrf
+            @include('status', ['errors' => $errors->all()])
+            @includeWhen(session('success'), 'toast', ['message' => session('success')])
+  <div class="card">
+    <div id="BankDetails">
```
Comment (LavishRawat) [inline resources/views/settings/bank-details.blade.php:11]: @AJAYSINGHBISHT12 Please fix the indentation.

File: resources/views/settings/bank-details.blade.php [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+      @csrf
```
Comment (LavishRawat) [inline resources/views/settings/bank-details.blade.php:12]: @AJAYSINGHBISHT12 why are we using @csrf here? can you please explain?

**Review by LavishRawat — CHANGES_REQUESTED** [review]

**Comment by GauravGusain98** [comment]
This is not needed

**Pull request closed without merging**